packages/twenty-front/src/testing/decorators/i18nFrontDecorator.tsx

```diff
@@ -8,7 +8,7 @@ i18n.load({
 });
 i18n.activate('en');
 
-export const i18nDecoratorFront = (Story: StoryFn) => {
+export const i18nFrontDecorator = (Story: StoryFn) => {
```

Comment:
Let's start with uppercase I: I18NFrontDecorator

Reply:
Fixed in followup PR! Thanks